Detections/SecurityEvent/GainCodeExecutionADFSViaSMB.yaml

```diff
@@ -0,0 +1,72 @@
+id: 12dcea64-bec2-41c9-9df2-9f28461b1295
+name: Gain Code Execution on ADFS Server via SMB + Remote Service/Scheduled Task
```

Comment:
I don't know if the UI will like the slash here, maybe just write out the work 'or'. 'Remote Service or Scheduled Task'